### PR TITLE
fix: fix the failing checks

### PR DESCRIPTION
**Pull request by OmaymaMahjoub**
## What?
Minor fix in the environment loop.
## Why?
The mypy checks are failing in all the PR because of this error:
```
mava/environment_loop.py:575: error: Unsupported operand types for - ("Array" and "int")
mava/environment_loop.py:575: note: Left operand is of type "Union[Any, Array]"
mava/environment_loop.py:578: error: Argument 1 to "int" has incompatible type "Union[Any, Array]"; expected "Union[str, bytes, SupportsInt, SupportsIndex, SupportsTrunc]"
```
## How?
Minor type edit in the count value in environment loop file

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "chex",
     # TODO Unpin JAX, once new rlax and acme version is released.
     # To fix cannot import name 'tree_multimap' from 'jax.tree_util'.
-    "jax<=0.3.15",
+    "jax==0.3.17",
```
Comment (siddarthsingh1) [inline setup.py:43]: Should the setup.py file use `<=` instead of `==` to reduce requirement version issues for local installations?

**Review by siddarthsingh1 — COMMENTED** [review]
Thanks for looking at this @OmaymaMahjoub. I just have one small question about the installation versions.

File: setup.py (hunk 1 above)
Comment (OmaymaMahjoub) [reply setup.py:43]: Yeah you are right I will bring it back to <=0.3.15

**Review by arnupretorius — APPROVED** [review]
Thanks @OmaymaMahjoub 🙂

**Review by AsadJeewa — APPROVED** [review]
Thanks @OmaymaMahjoub and @RuanJohn

**Review by KaleabTessera — APPROVED** [review]
Thanks @RuanJohn @OmaymaMahjoub !

**Pull request merged**